### PR TITLE
feat: connectionless credential offer

### LINKED ISSUE #270: Add option to create a credential offer not bound to a connection
So like send-offer but then without connection_id like we already have with proof requests

### PR DESCRIPTION
**Pull request by morrieinmaas**
closes #270

### CODE REVIEW

File: app/tests/e2e/test_issuer.py [hunk 1]
```diff
@@ -90,6 +89,73 @@ async def test_send_credential(
     assert_that(records).extracting("protocol_version").contains("v1", "v2")
 
 
+@pytest.mark.asyncio
+async def test_create_offer(
```
Comment (TimoGlastra) [inline app/tests/e2e/test_issuer.py:93]: Can we add an e2e tests that creates an offer, creates a connectionless oob invitation, and the other agent accepting the oob inviation, and then waiting for the offer-received webhook event for credentials?

It's quite a complex flow that would be good to have a test for

Comment (morrieinmaas) [reply app/tests/e2e/test_issuer.py:93]: I guess we can

Comment (morrieinmaas) [reply app/tests/e2e/test_issuer.py:93]: isn't there a test for this in oob e2e tests already?

Comment (morrieinmaas) [reply app/tests/e2e/test_issuer.py:93]: ah that one isn't connectionless nvm

**Review by TimoGlastra — APPROVED** [review]

**Pull request merged**